### ISSUE TITLE
Assertion fails

### ISSUE THREAD

**Issue by siirtola** [FINNGEN/CO2AnalysisModules#30]
Running mod_resultsVisualization_CohortDemographics as standalone fails in assertion:

```
> pathToCO2AnalysisModulesConfigYalm  <-  testthat::test_path("config/atlasDemo_CO2AnalysisModulesConfig.yml")
> options = list(launch.browser=FALSE, port = 5907)
> 
> browseURL(paste0("http://localhost:5907/?pathToResultsDatabase=", pathToResultsDatabase))
> run_app(pathToCO2AnalysisModulesConfigYalm, options = options)
Error in run_app(pathToCO2AnalysisModulesConfigYalm, options = options) : 
  Assertion on 'CO2AnalysisModulesConfig' failed: Must be of type 'list', not 'character'.
Called from: mstop("Assertion on '%s' failed: %s.", var.name, res, call. = sys.call(-2L))
Browse[1]> Q

```

**Comment by siirtola** [comment]
@javier-gracia-tabuenca-tuni , I don't know where this is coming from

**Comment by siirtola** [comment]
Fixed in #35
